src/http/client.rs

```diff
@@ -1358,11 +1358,17 @@ impl Http {
     }
 
     /// Kicks a member from a guild.
-    pub fn kick_member(&self, guild_id: u64, user_id: u64) -> Result<()> {
+    pub fn kick_member(&self, guild_id: u64, user_id: u64, reason: Option<&str>) -> Result<()> {
```

Comment:
This is a `pub` function, so this is a breaking change, is it not?

Reply:
It seems so.
Easily fixed by passing None as an additional argument

Reply:
Easily fixed, sure, but a breaking change nonetheless